app/src/main/res/layout/activity_login.xml

```diff
@@ -45,18 +45,43 @@
             android:layout_marginBottom="@dimen/spacing_margin"
             android:textSize="@dimen/lato_prompt_font_size" />
 
-        <EditText
-            android:id="@+id/phone_number_input"
-            android:background="@drawable/textinputborder"
-            android:focusable="true"
-            android:gravity="center"
-            android:hint="@string/phone_number_input_text"
-            android:inputType="phone"
-            android:layout_height="@dimen/text_input_height"
-            android:layout_width="wrap_content"
-            android:paddingStart="@dimen/edittext_horiz_padding"
-            android:paddingEnd="@dimen/edittext_horiz_padding"
-            android:textColor="@color/offBlack" />
+        <LinearLayout
+            android:layout_width="match_parent"
```

Comment:
i think we can leave this one as-is. if we wanted to do it absolutely right we'd probably want to make a style guide for it.